docs/en/setup/backend/backend-receivers.md

```diff
@@ -109,4 +110,21 @@ receiver_jaeger:
     gRPCPort: ${SW_RECEIVER_JAEGER_PORT:14250}
 ``` 
 
-NOTICE, Jaeger receiver is only provided in `apache-skywalking-apm-x.y.z.tar.gz` tar.
\ No newline at end of file
+NOTICE, Jaeger receiver is only provided in `apache-skywalking-apm-x.y.z.tar.gz` tar.
+
+## Opencensus receiver
+
+Opencensus receiver supports to ingest agent metrics by meter-system. OAP can load the configuration at bootstrap. 
+If the new configuration is not well-formed, OAP fails to start up. The files are located at `$CLASSPATH/oc-rules`.
```

Comment:
With the possibility of these files added continuously, should we consider to use `application.yml` to activate the files? A similar question about the fetcher too. Because these files are located in the image together.

Reply:
How about adding a name for which rule file? And activate them through the system env, like ALS analyzer.

Reply:
Make sense to me. Let me implement it in another PR. I touched too many files in this one. 

and I also want to introduce the `configuration` model to support dynamic rules update.